README.md

```diff
@@ -1,7 +1,43 @@
 # Somfy TaHoma
 
+<p align=center>
+    <a href="https://github.com/iMicknl/python-tahoma-api/actions"><img src="https://github.com/iMicknl/python-tahoma-api/workflows/CI/badge.svg"/></a>
+    <a href="https://github.com/psf/black"><img src="https://img.shields.io/badge/code%20style-black-000000.svg" /></a>
+</p>
+
 An updated and async version of the original [tahoma-api](https://github.com/philklei/tahoma-api) by [@philklei](https://github.com/philklei). The aim of this wrapper is to offer an easy to consume Python wrapper for the internal API's used by tahomalink.com.
 
 Somfy TaHoma has an official API, which can be consumed via the [Somfy-open-api](https://github.com/tetienne/somfy-open-api). Unfortunately only a few device classes are supported via the official API, thus the need for this wrapper.
 
 This package is written for the Home Assistant [ha-tahoma](https://github.com/iMicknl/ha-tahoma) integration, but could be used by any Python project interacting with Somfy TaHoma devices.
+
+# Development
+
+## Installation
```

Comment:
Dev on VSCode follows this method?

Reply:
Yes the installation part is not linked to any IDE. That's the workflow I usualy follow. You should be able to start from  scratch in 5 min only.

Reply:
I use PyCharm any ways, so method 2 is good enough for me!